websites/D/Di.FM/presence.ts

```diff
@@ -16,6 +16,12 @@ presence.on("UpdateData", () => {
 		presenceData.state =
 			document.querySelectorAll(".track-name")[0].textContent;
 		presenceData.smallImageKey = "play";
+
+		presenceData.largeImageKey = (<HTMLImageElement>(
+			document.querySelector(
+				"div > section.track-region.col > div > div.artwork > div > img"
+			)
+		)).src;
```

Comment:
```suggestion
		presenceData.largeImageKey = 
			document.querySelector<HTMLImageElement>(
				"div > section.track-region.col > div > div.artwork > div > img"
			).src;
```
Doesn't something like this work? (check it first)

Reply:
Checked, it does work